scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+if ! command -v github_changelog_generator > /dev/null; then
+	echo "No github_changelog_generator in \$PATH, install via 'gem install github_changelog_generator'."
+	exit 1
+fi
+
+if [ -z "$CHANGELOG_GITHUB_TOKEN" ]; then
+	printf "\nWARNING: No \$CHANGELOG_GITHUB_TOKEN in environment, set one to avoid hitting rate limit.\n\n"
+fi
+
+if [ -z "$SEMVER_TAG" ]; then
+	echo "You must set \$SEMVER_TAG to your desired release semver version."
+	exit 1
+fi
+
+github_changelog_generator -u fastly -p go-fastly \
+	--future-release $SEMVER_TAG \
+	--no-pr-wo-labels \
+	--no-author \
+	--base CHANGELOG_HISTORY.md \
+	--since "v1.14.0" \
```

Comment:
A test left in by accident? This shouldn't be hard coded.

Reply:
Oh wait, I think I get it, this is the first tag we started using the generator on?

Reply:
Exactly!